app/src/main/res/layout/login_fragment.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    >
+
+    <RelativeLayout
+        android:id="@+id/containerLogo"
+        android:layout_width="match_parent"
+        android:layout_height="260dp"
+        android:gravity="top"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        >
+
+        <ImageView
+            android:id="@+id/imageView"
```

Comment:
even if you don't use these ids, you may think a better id instead of imageView and imageView2

Reply:
I agree with this comment, the IDs have to be more descriptive, for example: woloxImage.